lib/ansible/inventory/manager.py

```diff
@@ -609,9 +609,11 @@ def subset(self, subset_pattern):
             # allow Unix style @filename data
             for x in subset_patterns:
                 if x[0] == "@":
```

Comment:
As in https://github.com/ansible/ansible/pull/59776, can we get an `IndexError` here? Not a blocker but since you are 'here', if you want to fix.